.ci/ldif/test.example.com.ldif

```diff
@@ -0,0 +1,3 @@
+dn: ou=test,dc=example,dc=com
+objectClass: organizationalUnit
```

Comment:
There should be a base of ```dc=Sales, dc=example,dc=com``` to have the possibility to check for subtree-searches.